sdk/deepbook-v3/src/utils/config.ts

```diff
@@ -177,7 +176,7 @@ export class DeepBookConfig {
 	}
 
 	// Getters
-	getCoin(key: CoinKey) {
+	getCoin(key: string) {
```

Comment:
Why are we removing the CoinKey type, this removes the type safety and convenient auto-completes from a lot of methods.

Reply:
I think for the initial release we can cover it in docs only, since we could be creating new pools on the fly as needed and the new sdk changes may not reflect immediately (unsure about the sdk release schedule). Once the list of pools is stable, I like making an sdk update using one of the other suggestions